lib/BuildSystem/BuildSystem.cpp

```diff
@@ -1146,11 +1146,20 @@ class ShellCommand : public ExternalCommand {
     for (const auto& it: env) {
       environment.push_back({ it.getKey(), it.getValue() });
     }
-    
+   
+    auto outputCallback = [](StringRef data) {
```

Comment:
Let's keep a FIXME on these callbacks (and the ones below) that we still need to figure out how to delegate this to the higher level client.